ui/app/templates/vault/cluster/access/leases/error.hbs

```diff
@@ -10,32 +10,30 @@
   {{model.message}}
 {{/unless}}
 
-<div class="box is-sideless is-fullwidth is-marginless">
-  {{#if (eq model.httpStatus 400)}}
+{{#if (eq model.httpStatus 400)}}
+  <AlertBanner
+    @type="danger"
+    @message="{{model.keyId}} is not a valid lease ID"
+  />
+{{else if (eq model.httpStatus 404)}}
+  <EmptyState
+    @title="No leases with that ID"
+    @message="Unable to find lease for the ID &quot;{{model.keyId}}&quot;. Try going back to the lookup and re-entering the ID."
+  >
+    {{#link-to "vault.cluster.access.leases" tagName="button" class="link"}}Back to lookup{{/link-to}}
```

Comment:
Can we do `class="button link"` without the `tagName` so it's an `a` tag that looks like a button?

Reply:
Sure, but it will need some adjustment to styles. Right now the link styles are set up to work with button elements but not the `.button` class. Since we're wanting to do new styles for these anyway though that should be easy